Sources/Extensions/SwiftStdlib/DictionaryExtensions.swift

```diff
@@ -34,6 +34,14 @@ public extension Dictionary {
         keys.forEach { removeValue(forKey: $0) }
     }
 
+    /// SwifterSwift: Remove a random element from the dictionary.
+    @discardableResult public mutating func removeRandomElement() -> Value? {
+        if let key = Array(keys).randomItem {
```

Comment:
Unfortunately you can't use `SwifterSwift` functions internally. The project is designed so that each function can be copy-pasted as-is into your app. The main reason is that the whole library itself can't safely be added as a dependency because of potential name clashes with other 3rd party libraries.

Reply:
Okay, I will adjust the code accordingly.